.github/workflows/ci.yml

```diff
@@ -16,19 +16,17 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        goVer: [1.16, 1.17]
+        goVer: ["^1.16", "^1.17"]
     steps:
       - name: Set up Go ${{ matrix.goVer }}
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v2
         with:
           go-version: ${{ matrix.goVer }}
-        id: go
 
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v2
 
       - name: Test via ${{ matrix.goVer }}
         env:
-          STREAM_CHAT_API_KEY: ${{ secrets.STREAM_CHAT_API_KEY }}
-          STREAM_CHAT_API_SECRET: ${{ secrets.STREAM_CHAT_API_SECRET }}
+          STREAM_KEY: ${{ secrets.STREAM_CHAT_API_KEY }}
```

Comment:
himm are we adding this v5 changelog?

Reply:
well, this is a quite new feature and I assume no one is using it yet: https://github.com/GetStream/stream-chat-go/commit/6cac452969917b7cbd2c8cfe34ca149b90046377#diff-4b667feae66c9d46b21b9ecc19e8958cf4472d162ce0a47ac3e8386af8bbd8cf

I think we can make an exception here and not handle this as a breaking change

Reply:
master is already breaking, just not forget adding into changelog